lib/hobbit.rb

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
@Taamir-Ransome as a side note, the term "initial commit" should be reserved for the first commit in a repo

Reply:
better name for this commit would be "Create hobbit class"